concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-528	528	Mich hungert, durstet.	Mich durstet.	I am thirsty.	3474	BE THIRSTY
+Steinthal-1875-1565-529	529	satt	satt	full (not hungry)	106	SATIATED
+Steinthal-1875-1565-530	530	kauen	kauen	to chew	321	CHEW
+Steinthal-1875-1565-531	531	schlucken	schlucken	to swallow	897	SWALLOW
+Steinthal-1875-1565-532	532	verdauen	verdauen	to digest		
+Steinthal-1875-1565-533	533	Mehl	Mehl	flour	1594	FLOUR
+Steinthal-1875-1565-534	534	Brot	Brot	bread	1368	BREAD
+Steinthal-1875-1565-535	535	Fleisch	Fleisch	meat	634	MEAT
+Steinthal-1875-1565-536	536	Brühe (Suppe)	Brühe (Suppe)	broth	1547	SOUP
+Steinthal-1875-1565-537	537	Brei (die üblichen Speisen)	Brei	mush	3651	MUSH (FOOD)
+Steinthal-1875-1565-538	538	Frühstück	Frühstück	breakfast	1322	BREAKFAST
+Steinthal-1875-1565-539	539	Hauptmahlzeit	Hauptmahlzeit	main meal		
+Steinthal-1875-1565-540	540	Abendbrot	Abendbrot	dinner	1833	DINNER (SUPPER)
+Steinthal-1875-1565-541	541	berauschendes Getränk	berauschendes Getränk	intoxicating drink	2050	ALCOHOL (DRINK)
+Steinthal-1875-1565-542	542	betrunken	betrunken	drunk	1463	DRUNK
+Steinthal-1875-1565-543	543	Tabak rauchen	Tabak rauchen	to smoke tobacco	1689	SMOKE (INHALE)
+Steinthal-1875-1565-544	544	schmecken	schmecken	to taste	21	TASTE (SOMETHING)
+Steinthal-1875-1565-545	545	Geschmack	Geschmack	taste	2190	TASTE
+Steinthal-1875-1565-546	546	süss	süss	sweet	717	SWEET
+Steinthal-1875-1565-547	547	bitter	bitter	bitter	887	BITTER
+Steinthal-1875-1565-548	548	sauer	sauer	sour	1906	SOUR
+Steinthal-1875-1565-549	549	salzig	salzig	salty	1091	SALTY
+Steinthal-1875-1565-550	550	fressen	fressen	to devour sth.		
+Steinthal-1875-1565-551	551	nagen	nagen	to gnaw	2706	GNAW
+Steinthal-1875-1565-552	552	Kleidungsstück (für den Kopf, Fuss, Leib, Aermel)	Kleidungsstück	garment	1895	CLOTHES
+Steinthal-1875-1565-553	553	Kleidungsstück (für den Kopf, Fuss, Leib, Aermel)	Kleidungsstück für den Kopf	head garment		
+Steinthal-1875-1565-554	554	Kleidungsstück (für den Kopf, Fuss, Leib, Aermel)	Kleidungsstück für den Fuss	foot garment		
+Steinthal-1875-1565-555	555	Kleidungsstück (für den Kopf, Fuss, Leib, Aermel)	Kleidungsstück für den Leib	upper garment		
+Steinthal-1875-1565-556	556	Kleidungsstück (für den Kopf, Fuss, Leib, Aermel)	Kleidungsstück für den Aermel	arm garment		
+Steinthal-1875-1565-557	557	sich anziehen	sich anziehen	to get dressed	814	DRESS UP
+Steinthal-1875-1565-558	558	Staatskleider anlegen, ausziehen	Staatskleider anlegen	to put on a festive dress		
+Steinthal-1875-1565-559	559	Staatskleider anlegen, ausziehen	Staatskleider ausziehen	to take off a festive dress		
+Steinthal-1875-1565-560	560	Kleider reinigen	Kleider reinigen	to clean clothes	3434	WASH (CLOTHES)
+Steinthal-1875-1565-561	561	waschen	waschen	to wash	1453	WASH
+Steinthal-1875-1565-562	562	trocknen	trocknen	to dry	1786	DRY UP
+Steinthal-1875-1565-563	563	schmutzig	schmutzig	dirty, unclean	1230	DIRTY
+Steinthal-1875-1565-564	564	rein	rein	clean	704	CLEAN
```

Comment:
Double mapping

Reply:
This seem to be two identical CLEANs as far as I can tell (Steinthal-1875-1565-499 and Steinthal-1875-1565-564)? So keeping it would make sense if we're fine with the double occurrence.

Reply:
Yes, I see no difference between them either.